src/app/Views/app-layout/app-layout.component.html

```diff
@@ -0,0 +1,23 @@
+<div>
+  <mat-toolbar class="header">
+  </mat-toolbar>
+</div>
+
+<mat-sidenav-container class="sidenav-container">
+  <mat-sidenav mode="side" opened position="start">
+
+    <mat-nav-list>
+      <button routerLink="dashboard" routerLinkActive="active-link" mat-raised-button type="button"
```

Comment:
do you think we can get do a loop here on a map of routes so we don't copy paste the same template?